### ISSUE TITLE
Input wav's time length for model "Wavegram_Logmel_Cnn14"?

### ISSUE THREAD

**Issue by machengnan** [qiuqiangkong/audioset_tagging_cnn#29]
Is there a requirement for Input wav's time length? 4s or 2s or any time?

I think this line code "x = torch.cat((x, a1), dim=1)" decide time length should be a certain value，right?

**Comment by qiuqiangkong** [comment]
The Wavegram_Logmel_Cnn14 is fixed to 10-second audio now. It is suggested to use Cnn14 which also performs well, and is flexible to any duration of audio clip.
